README.md

```diff
@@ -2,6 +2,17 @@
 
 ## Usage
 
+Ensure docker is running or if you are using Boot2Docker, run:
+
+```console
+boot2docker up
+boot2docker shellinit
```

Comment:
The boot2docker README does use the `eval` line without further explanation, so we should probably follow that.